ibis/backends/pandas/client.py

```diff
@@ -80,11 +80,26 @@
 
 
 _inferable_pandas_dtypes = {
-    'boolean': dt.boolean,
     'string': dt.string,
-    'unicode': dt.string,
     'bytes': dt.string,
-    'empty': dt.string,
+    'floating': dt.float64,
+    'integer': dt.int64,
+    'mixed-integer': dt.binary,
```

Comment:
Can you create a follow up to revisit the choice of binary for unsupported types?

Reply:
I'll create a follow up issue after this PR